src/components/PlioQuestion.vue

```diff
@@ -218,13 +227,88 @@ export default {
           this.hideSkipButton();
           this.isAnswerSubmitted = true;
           this.disableRadioButtons();
-        }, 200);
+        });
       }
 
       this.text = "";
       this.show = true;
       document.querySelector("body").classList.add("overflow-hidden");
       this.newProgressBarInfo['progressPercent'] = this.progressBarInfo['progressPercent']
+      this.handleImage()
+      this.styleQuestion()
+      this.styleOptions()
+      this.renderContent()
+    },
+
+    handleImage(){
+      this.$nextTick(() => {
```

Comment:
This method executes the code only after the DOM updation cycle has finished. Will only execute the code when everything in the DOM is ready and updated so we won't face any issues while running `document.getElementById` type of searches, hence, removing the need of a `setTimeOut` method

Reply:
Cool!